tools/hpo/comet_sweep.yaml

```diff
@@ -0,0 +1,15 @@
+algorithm: "bayes"
```

Comment:
It would be better if we move the config files to `tools/hpo/configs` and within that have `comt.yaml` and `wandb.yaml`